### PR TITLE
feat: Updates for Levski release

### PR DESCRIPTION
**Pull request by lenny-intel**
Signed-off-by: Leonard Goodell <leonard.goodell@intel.com>

**If your build fails** due to your commit message not passing the build checks, please review the guidelines here: https://github.com/edgexfoundry/edgex-compose/blob/main/.github/Contributing.md

## PR Checklist
Please check if your PR fulfills the following requirements:

- [x] I am not introducing a breaking change (if you are, flag in conventional commit message with `BREAKING CHANGE:` describing the break)
- [x] I have fully tested (add details below) this the new feature or bug fix (if not, why?)
- [ ] I have opened a PR for the related docs change (if not, why?)
  <link to docs PR>

### CODE REVIEW

**Review by jim-wang-intel — APPROVED** [review]
LGTM

File: README.md [hunk 1]
```diff
@@ -1,30 +1,19 @@
-## Edgex Docker Compose
-[![Build Status](https://jenkins.edgexfoundry.org/view/EdgeX%20Foundry%20Project/job/edgexfoundry/job/edgex-compose/job/main/badge/icon)](https://jenkins.edgexfoundry.org/view/EdgeX%20Foundry%20Project/job/edgexfoundry/job/edgex-compose/job/main/) [![GitHub License](https://img.shields.io/github/license/edgexfoundry/edgex-compose)](https://choosealicense.com/licenses/apache-2.0/) [![GitHub Pull Requests](https://img.shields.io/github/issues-pr-raw/edgexfoundry/edgex-compose)](https://github.com/edgexfoundry/edgex-compose/pulls) [![GitHub Contributors](https://img.shields.io/github/contributors/edgexfoundry/edgex-compose)](https://github.com/edgexfoundry/edgex-compose/contributors) [![GitHub Committers](https://img.shields.io/badge/team-committers-green)](https://github.com/orgs/edgexfoundry/teams/edgex-compose-committers/members) [![GitHub Commit Activity](https://img.shields.io/github/commit-activity/m/edgexfoundry/edgex-compose)](https://github.com/edgexfoundry/edgex-compose/commits)
```
Comment (bnevis-i) [inline README.md:2]: IMO the badges shouldn't have been deleted.

Comment (lenny-intel) [reply README.md:2]: They are still on the main branch README. Felt they made this README to busy.

**Review by bnevis-i — COMMENTED** [review]
Awaiting edgex-ui and device-rest to be pushed before validating the config to see if it boots.

**Review by bnevis-i — APPROVED** [review]
Smoke tested. LGTM.

**Comment by lenny-intel** [comment]
> Smoke tested. LGTM.

Thanks @bnevis-i !

**Pull request merged**